README.md

```diff
@@ -131,7 +131,7 @@ Metrics will then be available from `<application-root>/metrics`.
 
 You can find all the releases on the [Releases page](https://github.com/oracle/weblogic-monitoring-exporter/releases/).
```

Comment:
You can find all the releases -> You can all the exporter releases (just for the sake of clarity)